### PR TITLE
Update Chinese (Taiwan) Translation.

### PR DESCRIPTION
**Pull request by pan93412**

### CODE REVIEW

**Comment by pan93412** [comment]
@slacka

File: src/locale/zh_TW/LC_MESSAGES/woeusb.po [hunk 1]
```diff
@@ -13,137 +14,152 @@ msgstr ""
...
 msgid "Error..."
-msgstr "錯誤……"
+msgstr "錯誤…"
 
-#: ../../../App.cpp:117
-msgid "Unknown error as occurred!"
-msgstr "發生了未知的錯誤！"
+#: ../App.cpp:117
+msgid "Unknown error as occured!"
+msgstr "發生了未知錯誤！"
```
Comment (PeterDaveHello) [inline src/locale/zh_TW/LC_MESSAGES/woeusb.po]: This isn't seem to be needed IMO

File: src/locale/zh_TW/LC_MESSAGES/woeusb.po [hunk 2]
```diff
@@ -1,9 +1,10 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: winusb\n"
-"Report-Msgid-Bugs-To: WoeUSB Issue Tracker <https://github.com/slacka/WoeUSB/issues>\n"
-"POT-Creation-Date: 2011-07-14 00:08+0100\n"
-"PO-Revision-Date: 2016-07-09 20:51+0800\n"
+"Report-Msgid-Bugs-To: WoeUSB Issue Tracker <https://github.com/slacka/WoeUSB/"
+"issues>\n"
```
Comment (PeterDaveHello) [inline src/locale/zh_TW/LC_MESSAGES/woeusb.po]: Why is here wrapped?

File: src/locale/zh_TW/LC_MESSAGES/woeusb.po [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
+"POT-Creation-Date: 2011-07-14 00:07+0100\n"
```
Comment (PeterDaveHello) [inline src/locale/zh_TW/LC_MESSAGES/woeusb.po]: Should this line be touched?

File: src/locale/zh_TW/LC_MESSAGES/woeusb.po [hunk 4]
```diff
@@ -13,137 +14,152 @@ msgstr ""
...
-#: ../../../MainPanel.cpp:53
+#: ../MainPanel.cpp:53
 msgid "Please select a disk image"
 msgstr "請選取一個光碟映像檔"
 
-#: ../../../MainPanel.cpp:58
+#: ../MainPanel.cpp:58
 msgid "From a CD/DVD drive"
-msgstr "從一個 CD/DVD 光碟機"
+msgstr "從 CD/DVD 光碟機裝置"
```
Comment (PeterDaveHello) [inline src/locale/zh_TW/LC_MESSAGES/woeusb.po]: I prefer to remove the additional last two chars here

File: src/locale/zh_TW/LC_MESSAGES/woeusb.po [hunk 5]
```diff
@@ -13,137 +14,152 @@ msgstr ""
...
+#: ../MainPanel.cpp:247
 msgid "Please wait..."
-msgstr "請稍候……"
+msgstr "請稍候…"
 
-#: ../../../MainPanel.cpp:270 ../../../MainPanel.cpp:289
+#: ../MainPanel.cpp:270 ../MainPanel.cpp:289
 msgid "Are you sure you want to cancel the installation?"
-msgstr "請問您確定要取消安裝程序嗎？"
+msgstr "確定取消安裝程序？"
```
Comment (PeterDaveHello) [inline src/locale/zh_TW/LC_MESSAGES/woeusb.po]: Both version look a little bit not so smooth 😅

File: src/locale/zh_TW/LC_MESSAGES/woeusb.po (hunk 3 above)
Comment (pan93412) [reply src/locale/zh_TW/LC_MESSAGES/woeusb.po]: Oh, that is modified by my PO editor, just ignore those.

**Comment by pan93412** [comment]
@PeterDaveHello, could you have a look? :)

**Comment by Lin-Buo-Ren** [comment]
I would like to cherry-pick some of the changes instead of merging all of them.  Thanks for your understanding.

**Pull request closed without merging**